tests/testthat/test-monitor.R

```diff
@@ -0,0 +1,113 @@
+describe("vetiver_compute_metrics()", {
+
+    library(dplyr)
+    library(parsnip)
```

Comment:
Jenny's talk on Monday would discourage the usage of `library()` anywhere inside the `testthat/` folder. Instead we'd just use `dplyr::` or `parsnip::`

Being inside the context set up by `describe()` means the objects like `lm_aug` are locally scoped, which is great, but the library calls will still affect other test files because they are global changes